fedimint-api-client/src/api/iroh.rs

```diff
@@ -92,20 +111,77 @@ impl IClientConnector for IrohConnector {
             .get(&peer_id)
             .ok_or(PeerError::InvalidPeerId { peer_id })?;
 
-        let connection = match self.connection_overrides.get(&node_id) {
-            Some(node_addr) => {
-                trace!(target: LOG_NET_IROH, %node_id, "Using a connectivity override for connection");
-                self.endpoint
-                    .connect(node_addr.clone(), FEDIMINT_API_ALPN)
-                    .await
+        let mut join_set = JoinSet::new();
```

Comment:
Nice use of a `JoinSet` :+1:

Reply:
It compiles? Any reason it wouldn't work?

Reply:
I haven't verified by running, but I don't see anything different with `JoinSet` that would cause a wasm issue compared to our other `tokio` goodies. I think it's just using `tokio::spawn` and `JoinHandle` under the hood, which we're already able to use with wasm. I'm curious if I'm missing something though @maan2003 

https://github.com/tokio-rs/tokio/blob/master/tokio/src/task/join_set.rs